### PR TITLE
[NET 6 Attribute Conversion] Foundation

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

**Review by mandel-macaque — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]
👍

File: src/Foundation/NSMetadataItem.cs [hunk 1]
```diff
@@ -242,7 +262,13 @@ bool GetBool (NSString key)
 #if NET
 		public bool? UbiquitousItemDownloadRequested {
 #else
-		[iOS (8,0)][Mac (10,10)]
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
```
Comment (rolfbjarne) [inline src/Foundation/NSMetadataItem.cs:266]: This is inside a !NET block from the #if NET just above, so it won't do the right thing. These attributes should be moved into the block above instead.

File: src/Foundation/NSMetadataItem.cs [hunk 2]
```diff
@@ -257,7 +283,13 @@ bool GetBool (NSString key)
 #if NET
 		public bool? UbiquitousItemIsExternalDocument {
 #else
-		[iOS (8,0)][Mac (10,10)]
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
```
Comment (rolfbjarne) [inline src/Foundation/NSMetadataItem.cs:287]: Same.

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — APPROVED** [review]

**Comment by chamons** [comment]
Rekicking due to possibly flaky tests

**Comment by chamons** [comment]
https://github.com/xamarin/maccore/issues/2443 strikes again.

**Pull request merged**